syncmaster/worker/spark.py

```diff
@@ -37,7 +37,12 @@ def get_worker_spark_session(
 
 def get_packages(db_type: str) -> list[str]:
     from onetl.connection import MSSQL, Clickhouse, MySQL, Oracle, Postgres, SparkS3
-    from onetl.file.format import Excel
+    from onetl.file.format import XML, Excel
+
+    # see supported versions from https://mvnrepository.com/artifact/com.crealytics/spark-excel
+    file_formats_spark_packages: list[str] = XML.get_packages(spark_version="3.5.1") + Excel.get_packages(
```

Comment:
I don't get why spark_version is hardcoded for XML

Reply:
For Excel you can update URL to https://github.com/nightscape/spark-excel/issues/902

Reply:
I don't mean make some changes to `spark_version="..."` format. I mean just add a comment why Spark version is hardcoded here, linking this issue.